conversation_simulator/src/states/Boot.js

```diff
@@ -36,7 +36,15 @@ export default class extends Phaser.State {
   }
 
   preload () {
-    this.load.json('gameData', 'assets/data/' + gameModule + '.json')
+    if (typeof game.gameModule === 'string') {
+      let gameData = this.load.json('gameData', 'assets/data/' + game.gameModule + '.json')
+      gameData.onLoadComplete.add(function () {
+        game.gameModule = this.game.cache.getJSON('gameData')
+        if (game.gameModule === null) {
+          alert('Could not load module')
```

Comment:
let's not do alerts. we should show text in UI

Reply:
Text now shows up on screen instead of an alert.